starmeta/semanticdb/shared/src/main/protobuf/semanticdb.proto

```diff
@@ -11,19 +11,20 @@ message Attributes {
   reserved 5;
   string filename = 9;
   string contents = 8;
-  string dialect = 7;
+  string language = 7;
   repeated ResolvedName names = 2;
   repeated Message messages = 3;
-  repeated SymbolDenotation denotations = 4;
+  repeated ResolvedSymbol symbols = 4;
   repeated Sugar sugars = 6;
 }
 
 message ResolvedName {
-  Range range = 1;
-  string symbol = 2;
+  Position pos = 1;
+  string sym = 2;
+  bool is_binder = 3;
```

Comment:
This simplifies metaide very very much and is almost mandatory when semanticdb becomes polyglot (because then it is not easy to have a parser at hand for everything that you read).